tests/Unit/ControlSystem/Test_Averager.cpp

```diff
@@ -248,6 +251,28 @@ SPECTRE_TEST_CASE("Unit.ControlSystem.Averager.BadCallToAverageTime",
   averager.average_time(0.0);
 }
 
+SPECTRE_TEST_CASE("Unit.ControlSystem.Averager.EqualityAndSerialization",
+                  "[ControlSystem][Unit]") {
```

Comment:
Change the test cases in this file into functions and call them all from one test case.  We realized at some point that each test case has a significant overhead on some clusters.